src/extension.ts

```diff
@@ -105,9 +106,19 @@ export function activate(context: vscode.ExtensionContext) {
 
         const showVulnerabilityFoundPrompt = async (msg: string) => {
           const selection = await vscode.window.showWarningMessage(`${msg}. Powered by [Snyk](${registrationURL})`, StatusMessages.FULL_STACK_PROMPT_TEXT);
+          // telemetry
+          let event = {
+            type: 'track', // type of telemetry event such as : identify, track, page, etc.
+            name: '',
+          }
           if (selection === StatusMessages.FULL_STACK_PROMPT_TEXT) {
             vscode.commands.executeCommand(Commands.TRIGGER_FULL_STACK_ANALYSIS);
+            event.name = reportOpenedNotificationPopup;
+          }
+          else {
+            event.name = notificationPopupDismissed;

```

Comment:
In that case, we can probably change the event to something like:

```
{
name:'vulnerability_report_popup',
opened: true/false
}
```

Reply:
done